runtime/plugin/code_actions.lua

```diff
@@ -0,0 +1,46 @@
+local vscode = require("vscode-neovim")
+
+local k = function(mode, lhs, rhs)
```

Comment:
I don't think this helper provides enough value to keep users from seeing the full details in one line.

Reply:
This is concise and common

Reply:
It's just if users are supposed to use the file as a reference for the API, it should be simple and straightforward instead of needing to also copy a helper. It is nice if users can just copy/paste directly.

Reply:
This is the most common operation, and users can see the most critical "expr" parameter at a glance. There is no misleading anywhere. A user who just wants to copy and paste will not look at these files at all.